src/modules/App.js

```diff
@@ -6,7 +6,8 @@ const AppLayout = React.lazy(() => import('containers/AppLayout'));
 const Login = React.lazy(() => import('modules/login/Loginv1'));
 const ForgotPassword = React.lazy(() => import('modules/forgot-password/Forgot-Password'));
 const Page404 = React.lazy(() => import('modules/placeholder/page404/Page404'));
-const Calendar = React.lazy(() => import('modules/calendar/Calendar'));
+// const Calendar = React.lazy(() => import('modules/calendar/Calendar'));
+const Todo = React.lazy(() => import('views/test/fileupload'));
```

Comment:
@LloryGlaus422  pls rename files and variables accordingly to their functions , thanks

Reply:
please ko @LloryGlaus422  I need it na